flag.go

```diff
@@ -683,5 +715,15 @@ func stringifySliceFlag(usage, name string, defaultVals []string) string {
 	}
 
 	usageWithDefault := strings.TrimSpace(fmt.Sprintf("%s%s", usage, defaultVal))
-	return fmt.Sprintf("%s\t%s", prefixedNames(name, placeholder), usageWithDefault)
+	return fmt.Sprintf("%s\t%s", prefixedNames(names, placeholder), usageWithDefault)
+}
+
+func hasFlag(flags []Flag, fl Flag) bool {
```

Comment:
Whoops!  Thanks for catching that :smile_cat:

On Sat, May 21, 2016, 22:08 Jesse Szwedko <notifications@github.com> wrote:

> In flag.go
> <https://github.com/codegangsta/cli/pull/419#discussion_r64142989>:
>
> > @@ -683,5 +715,15 @@ func stringifySliceFlag(usage, name string, defaultVals []string) string {
> >  	}
> >
> >  	usageWithDefault := strings.TrimSpace(fmt.Sprintf("%s%s", usage, defaultVal))
> > -	return fmt.Sprintf("%s\t%s", prefixedNames(name, placeholder), usageWithDefault)
> > +	return fmt.Sprintf("%s\t%s", prefixedNames(names, placeholder), usageWithDefault)
> > +}
> > +
> > +func hasFlag(flags []Flag, fl Flag) bool {
>
> Is this called anywhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/codegangsta/cli/pull/419/files/81fcf706ea1c2bdde52d04088a9c5b0bc1a97f7a#r64142989>
>